main.js

```diff
@@ -0,0 +1,147 @@
+const VALIDATE = (prop) => {
+    if (prop === undefined || prop === null) {
+        return true;
+    }
+    return false;
+}
+
+// 1. Cтворити функцію, яка створює та реєструє нову команду у автопробізі. Під час виклику функції потрібно передати дані про команду - назва команди, ім'я водія, рік народження водія, марка автомобіля, спонсор команди та чи водій допущений до автопробігу
+
+// 1st var
+const CAR_RACE_TEAM = (teamName, driverName, yearOfBirth, carBrand, sponsor, isAllowed) => {
+    if (VALIDATE(teamName) || VALIDATE(driverName) || VALIDATE(yearOfBirth) || VALIDATE(carBrand) || VALIDATE(sponsor) || VALIDATE(isAllowed) || isNaN(yearOfBirth)) {
+        console.log("Something went wrong! Check the information and try to fill one more time.");
+    } else {
+        const NEW_TEAM = {
+            teamName: teamName,
+            driverName: driverName,
```

Comment:
As an option, you can put driver info in a nested object

```suggestion
            driver: {
                name: driverName,
                yearOfBirth: yearOfBirth,
            }
            
```